shared/stateutil/state_root_test.go

```diff
@@ -82,7 +75,7 @@ func BenchmarkHashTreeRootState_Generic_512(b *testing.B) {
 	genesisState := setupGenesisState(b, 512)
 	b.StartTimer()
 	for i := 0; i < b.N; i++ {
-		if _, err := stateutil.HashTreeRootState(genesisState); err != nil {
+		if _, err := ssz.HashTreeRoot(genesisState); err != nil {
```

Comment:
These 3 tests here say `BenchmarkHashTreeRootState` when it's `HashTreeRoot` being tested instead, can you update the test names?

Reply:
Fixed